lib/js_of_ocaml/dom_html.ml

```diff
@@ -747,6 +753,18 @@ and element = object
 
   method blur : unit meth
 
+  method hidePopover : unit meth
+
+  method showPopover : unit meth
+
+  method showPopover_options : _ -> unit meth
+
+  method togglePopover : bool t meth
+
+  method togglePopover_force : bool t -> bool t meth
+
+  method togglePopover_options : _ -> bool t meth
+
```

Comment:
[According to the specs](https://html.spec.whatwg.org/multipage/dom.html#showpopoveroptions), these methods accept a dictionary as a parameter for passing options. How would that be reflected in Js_of_ocaml?